src/d3a/d3a_core/redis_connections/redis_communication.py

```diff
@@ -111,9 +112,18 @@ def publish_results(self, endpoint_buffer):
             return
         result_report = endpoint_buffer.generate_result_report()
         results_validator(result_report)
+
         results = json.dumps(result_report)
+        message_size = utf8len(results)
+        if message_size > 64000:
+            log.error(f"Do not publish message bigger than 64 MB, current message size "
+                      f"{message_size / 1000.0} MB.")
+            return
```

Comment:
So what happens here? Te data package that is too large, will not be sent, so there will be a whole in the results on the d3a-web or d3a-ui side, right?

Reply:
Exactly. The idea is to not stop the simulation, but to send an error message to Kibana/Sentry, and simply the UI results will have gaps from the data that are missing.